kratos/utilities/force_and_torque_utils.cpp

```diff
@@ -100,16 +100,16 @@ std::array<array_1d<double,3>,2> ForceAndTorqueUtils::ComputeEquivalentForceAndT
     // Check whether nodes have the torque variable and set the lambda accordingly.
     // Note: multiple node types within the model part are not supported.
     //       In that case, this check would have to performed for each node.
-    if (rModelPart.GetCommunicator().LocalMesh().Nodes().begin()->SolutionStepsDataHas(rTorqueVariable)) {
+    if (rModelPart.HasNodalSolutionStepVariable(rTorqueVariable)) {
```

Comment:
FYI @matekelemen those checks easily fail in MPI when there are partitions that don't have locals nodes

Reply:
Ouch, that's a residue of a bug hunt I forgot about, sorry.